File: Calci/app/src/main/java/com/dharwin/MainActivity.java
package com.dharwin;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.view.View;
import android.widget.Button;
import android.widget.EditText;
import android.widget.TextView;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        final EditText numa=findViewById(R.id.num1);
        final EditText numb=findViewById(R.id.num2);
        final TextView te=findViewById(R.id.textView);
        Button b1=findViewById(R.id.button);
        b1.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                int n1=Integer.parseInt(numa.getText().toString());
                int n2=Integer.parseInt(numb.getText().toString());
                te.setText(Integer.toString(n1+n2));
            }
        });
        Button b2=findViewById(R.id.button2);
        b2.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                int n1=Integer.parseInt(numa.getText().toString());
                int n2=Integer.parseInt(numb.getText().toString());
                te.setText(Integer.toString(n1-n2));
            }
        });
        Button b3=findViewById(R.id.button3);
        b3.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                int n1=Integer.parseInt(numa.getText().toString());
                int n2=Integer.parseInt(numb.getText().toString());
                te.setText(Integer.toString(n1*n2));
            }
        });
    }
}

File: ChangeFontAndColor/app/src/main/java/com/example/myapplication/MainActivity.kt
package com.example.myapplication

import android.graphics.Color
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.widget.Button
import android.widget.TextView

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        val b1 = findViewById(R.id.button) as Button;
        val b2 = findViewById(R.id.button2) as Button;
        val text = findViewById(R.id.text) as TextView;
        var ch = 1;
        var font = 20f;
        b1.setOnClickListener() {
            text.setTextSize(font);
            font = font + 5;
            if (font == 50f) {
                font = 30f;

            }
        }
        b2.setOnClickListener() {
            when (ch) {
                1 -> text.setTextColor(Color.parseColor("#FF0000"));
                2 -> text.setTextColor(Color.parseColor("#E9F542"));
                3 -> text.setTextColor(Color.parseColor("#4B42F5"));
                4 -> text.setTextColor(Color.parseColor("#F542F2"));
                5 -> text.setTextColor(Color.parseColor("#F54275"));
            }
            ch++;
            if (ch == 5) {
                ch = 1;
            }

        }
    }
}